stable/kommander-karma/README.md

```diff
@@ -32,7 +32,7 @@ karma:
       traefik.ingress.kubernetes.io/auth-type: "forward"
       traefik.ingress.kubernetes.io/auth-url: "http://traefik-forward-auth-kubeaddons.kubeaddons.svc.cluster.local:4181/"
       traefik.ingress.kubernetes.io/priority: "2"
-    path: "/ops/portal/kommander/monitoring/karma/"
+    path: "/ops/portal/kommander/monitoring/karma"
```

Comment:
wasn't this needed bc there was a bug of some sort? or this was due to the kommander-ui hijacking?

Reply:
there is another bug with the slash but we were able to sidestep it via our traefik/karma settings